api/room/mutations.js

```diff
@@ -9,6 +9,12 @@ import { pubsub } from "../graphql/pubsub.js";
 
 export default {
   addRoom: async (_, { userid, cards }) => {
+
+    // assign a default value ONLY when cards array is empty
+    if (cards == undefined || cards == null) {
+      cards = [1, 2, 3, 5, 8, 13, 20, 1000]
```

Comment:
Set cards to an empty array if undefined or null.
We ran into issues in the past where arrays aren't being passed into the server correctly. Having it set as the default values would make debugging a nightmare